WBooks/Components/Detail/DetailViewModel.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  DetailViewModel.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 30/08/21.
+//
+
+import Foundation
+
+class DetailViewModel {
+    
+    let book: Book
+    
+    init(book: Book) {
+        self.book = book
+    }
+    
+    func isAvaibleRent() -> Bool {
+        return book.status == "Available"
+    }
+    
+    func rent(action: @escaping (Error?) -> () ) {
+        
+        let onSuccess = { (rent: Rent) in
+            action(nil)
+        }
+                
+        let onError = { error in
+            action(error)
+        }
+        
+        let repository = RentRepository()
+        repository.setRent(parameters: getParametersRent(), onSuccess: onSuccess, onError: onError)
+    }
+    
+    func getParametersRent() -> Rent {
+        
+        let today = Date()
+        let tomorrow = Calendar.current.date(byAdding: .day, value: 1, to: today)!
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd"
+        
+        let parameters = Rent(
+            userId: 10,
```

Comment:
Why `10`?

Reply:
In the card it said that the trainer should be consulted for the user id to manage, that id was given to me by Gabi, I guess it is because there is no functional login, then the user id is burned.

Reply:
In that case, we can create a constant like` let userId = 10` to be more clear